.github/workflows/build-android.yml

```diff
@@ -0,0 +1,60 @@
+name: Build Rust
```

Comment:
Shouldn't this be something like "Build Android"?

Reply:
I suppose. The top-level name gets stripped out anyway in favor of the calling job's name, so I don't think it matters very much.

Reply:
This particular top-level name gets stripped out because it is part of a reusable workflow.

Reply:
Turns out we don't! So I've removed it.